File: Java/src/secao06/Exercicio04.java
package secao06;

import java.util.Scanner;

public class Exercicio04 {
    public static void main(String[] args) {
        // var
        float altura, peso_ideal;
        char sexo;
        Scanner teclado = new Scanner(System.in);

        // input
        System.out.println("Informe sua altura:");
        altura = teclado.nextFloat();
        
        System.out.println("Informe seu sexo m/f:");
        sexo = teclado.next().charAt(0);

        // processing
        if (sexo == 'm') {
            peso_ideal = (float)(72.7 * altura) - 58;
            System.out.println("Seu peso ideal é " + peso_ideal);
        }
        if (sexo == 'f') {
            peso_ideal = (float)(62.1 * altura) - 58;
            System.out.println("Seu peso ideal é " + peso_ideal);
        }
        if (sexo != 'm' && sexo != 'f'){
            System.out.println("Sexo não reconhecido.");
            peso_ideal = 0;
        }
        teclado.close();
    }
}

File: Python/secao03/Exercicio04.py
# input
altura = float(input("Informe sua altura: "))
sexo = input("Informe o sexo m/f: ")

# processing
if sexo.lower() == 'm':
    peso_ideal = (72.7 * altura) - 58
elif sexo.lower() == 'f':
    peso_ideal = (62.1 * altura) - 58
else:
    peso_ideal = 0
    print("Sexo não reconhecido.")

# output
print("Seu peso ideal é {0:.2f}".format(peso_ideal))